### DISCUSSION TITLE
Installing from scratch on Ubuntu server: manually setting up everything

### DISCUSSION THREAD

**Discussion by akelai** [nocodb/nocodb#550] [Discord Threads]
I understand the convenience of Docker, but I miss instructions that could help installing from scratch on Ubuntu server, for a production environment, without using Docker and with the awareness of every piece - I mean something like "`make this directory`", "`set these permissions`", "`set up a PostgreSQL database with a user`", "`set up Redis`", "`sudo something`", "`configure NGINX like this`", etc.

Can authors help on this by writing these instructions, possibly using some other open source project installation instructions as a good template and starting point?

Or, can someone here try to reach a working setup by installing from scratch (manually) everything, and share the instructions?

**Comment by o1lab** [comment]
We would love a pull request,
